src/portal_visualization/builders/base_builders.py

```diff
@@ -29,9 +48,15 @@ def __init__(self, entity, groups_token, assets_endpoint, **kwargs):
         self._files = []
 
     @abstractmethod
-    def get_conf_cells(self, **kwargs):  # pragma: no cover
+    def get_configs(self):  # pragma: no cover
         raise NotImplementedError()
 
+    def get_configs_cells(self, marker=None):
+        kwargs = {'marker': marker} if marker is not None else {}
+        configs = self.get_configs(**kwargs)
+        cells = _get_cells_from_conf_list(configs)
+        return ConfigsCells(configs, cells)
```

Comment:
This is the new base class method that constructs the list of cells and returns a tuple.